commons-jetty/src/main/scala/com/avsystem/commons/jetty/rpc/JsonJettyRPCFramework.scala

```diff
@@ -2,29 +2,20 @@ package com.avsystem.commons
 package jetty.rpc
 
 import com.avsystem.commons.serialization.GenCodec
-import upickle.Js
+import com.avsystem.commons.serialization.json.{JsonStringInput, JsonStringOutput}
 
 object JsonJettyRPCFramework extends JettyRPCFramework {
-  type RawValue = Js.Value
+  type RawValue = String
   type Reader[T] = GenCodec[T]
   type Writer[T] = GenCodec[T]
   type ParamTypeMetadata[T] = ClassTag[T]
   type ResultTypeMetadata[T] = DummyImplicit
 
-  def read[T: Reader](raw: RawValue): T = GenCodec.read[T](new JsValueInput(raw))
-  def write[T: Writer](value: T): RawValue = JsValueOutput.write[T](value)
+  def read[T: Reader](raw: RawValue): T = JsonStringInput.read[T](raw)
+  def write[T: Writer](value: T): RawValue = JsonStringOutput.write[T](value)
 
-  override protected def valueToJson(value: RawValue): String = upickle.json.write(value)
-  override protected def jsonToValue(json: String): Js.Value = upickle.json.read(json)
-  override protected def argsToJson(argLists: List[List[RawValue]]): String =
-    upickle.json.write(Js.Arr(argLists.map { args => Js.Arr(args: _*) }: _*))
-  override protected def jsonToArgs(json: String): List[List[Js.Value]] = {
-    upickle.json.read(json) match {
-      case Js.Arr(lists@_*) =>
-        lists.collect {
-          case Js.Arr(args@_*) => args.toList
-        }(scala.collection.breakOut)
-      case _ => Nil
-    }
-  }
+  override protected def valueToJson(value: RawValue): String = value
```

Comment:
Can we just drop these?

Reply:
If we decide that for all JettyRPCFrameworks `RawValue` = `String`.

Reply:
![rzeczywiście](https://vignette.wikia.nocookie.net/nonsensopedia/images/2/25/No_kurwa_rzeczywi%C5%9Bcie.jpeg/revision/latest?cb=20120921183029)

Reply:
But seriously, I think if in the end we have to convert `RawValue` to `String` then there's no point in keeping this redundant intermediate abstract type.